plugins/Insights/lang/en.json

```diff
@@ -1,35 +1,35 @@
 {
     "Insights": {
-        "PluginDescription": "Provides Insights about your traffic. Insights are available as dashboard widgets as well as a new icon in reports, letting you see most important trends in your data.",
+        "PluginDescription": "Provides traffic insights. Browse your most important data trends from dashboard widgets or an icon in the reports.",
         "ControlComparedToDescription": "Growth compared to the",
-        "ControlFilterByDescription": "Show all, only movers, only new or only disappeared",
-        "DatePeriodCombinationNotSupported": "It is not possible to generate insights for this date and period combination.",
+        "ControlFilterByDescription": "Show all, only movers, only new, or only disappeared",
+        "DatePeriodCombinationNotSupported": "Cannot generate insights for this date over this period.",
         "DayComparedToPreviousDay": "previous day",
-        "DayComparedToPreviousWeek": "same day in the previous week",
-        "DayComparedToPreviousYear": "same day in the previous year",
+        "DayComparedToPreviousWeek": "same day the week before",
+        "DayComparedToPreviousYear": "same day the year before",
         "Filter": "Filter",
-        "FilterIncreaserAndDecreaser": "Increaser & decreaser",
-        "FilterOnlyDecreaser": "Only decreaser",
+        "FilterIncreaserAndDecreaser": "Increasing and decreasing",
+        "FilterOnlyDecreaser": "Only decreasing",
         "FilterOnlyDisappeared": "Only disappeared",
-        "FilterOnlyIncreaser": "Only increaser",
+        "FilterOnlyIncreaser": "Only increasing",
         "FilterOnlyMovers": "Only movers",
         "FilterOnlyNew": "Only new",
         "IgnoredChanges": "Changes affecting less than %s visits were ignored.",
         "MonthComparedToPreviousMonth": "previous month",
-        "MonthComparedToPreviousYear": "same month in the previous year",
-        "MoversAndShakersWidgetTitle": "Movers and Shakers",
-        "NoResultMatchesCriteria": "No rows match the criteria",
+        "MonthComparedToPreviousYear": "same month the year before",
+        "MoversAndShakersWidgetTitle": "Movers and shakers",
+        "NoResultMatchesCriteria": "There are no such rows",
         "OverviewWidgetTitle": "Insights Overview",
-        "TitleConsideredInsightsChanges": "The rows increased or decreased by at least %1$s visits (%2$s%% of %3$s total visits).",
-        "TitleConsideredInsightsGrowth": "The following rows have a growth of at least %1$s%% compared to %2$s.",
-        "TitleConsideredMoversAndShakersChanges": "Considered movers only if they grew by more than %1$s%% visits or shrank by less than %2$s%% visits, new entries only if they increased by more than %3$s%% visits (%4$s), and disappeared rows if they shrank by less than %5$s%% visits (%6$s).",
```

Comment:
If it shrank by fewer (as opposed to more) than a certain number of visits, I don't see how it is a mover… (Changed meaning in the proposal.)

Reply:
Actually I don't really understand what the text is trying to say. Having it filled with numbers looks like this:
![image](https://user-images.githubusercontent.com/1579355/176390024-7196f837-605a-4b81-a1a8-7fc51ea4e78b.png)

IIRC @tsteur build this feature. Maybe he can say more on that.

Reply:
Wouldn't they need to shrink by more than 84%? As if it only shrank by 60%, then it's not a significant change since the visits also shrank by 60%.

Also maybe it should say `and disappeared rows if shrinking by more than %5$s%% (%6$s).`?

Generally, movers and shakers would only show rows that changed significantly. Or only shows new or disappeared rows when they have/had at least a certain amount of visits. But we wouldn't show it if a row with little traffic disappeared.

Not sure if any of that helps?

Reply:
Maybe it could be worked out from the numbers, but other than that it just gets harder and harder to understand.

So it is a mover if it is in the 84.1th percentile, and also if it shrank by less than a negative of the total, which is to say it is at the bottom 15,9th percentile for growth?
Or, since the average is -64,1, it has to rank below the Q1 H-spread at -84% and below.
That isn't really what "less than" means though…
Something is missing between "n% ___ visits". Of?
"New" entries are the ones that didn't make the cut-off because they previously were seen by too few people (?), and 5% is either where that puts it, the number of visits 117 amounts to, or it concerns 117 pages…?
The consideration to go missing is to post a greater decline "less than"-than the bottom 7%(?)
and this concerns either (164) unique visits, or pages.

Also:
**Date sequence**
"Mar 2022" follows 2335, when it could be
Unique page-views changed from dropped from 6510 in Mar of 2022 to 2335 (when?).
**Number behind movers**
Whatever (number) is, why isn't it there for movers?
**Line shifting**
One sentence per line would help
**Order of categories**
"Disappeared" should maybe come before "New", since the qualification for going missing is the reasoning for coming back from disappearance(?) Don't know if that would help.

Reply:
> Is it something like moving more than the average increase, because otherwise all movers and shakers could just happen to be above the threshold when all pages are more or less evenly visited?

Exactly. If visits increase say 60%, then it looks at rows that are increasing say more than 80%.

> "New" entries are the ones that didn't make the cut-off because they previously were seen by too few people (?), and 5% is either where that puts it, the number of visits 117 amounts to as a percentage-wise gain, or it concerns 117 pages…?

It shows new pages when the row (label) wasn't visible in the previous period and now the row is there. And the row's visits is 5% of the total amount of visits. 117/2335= 5%. If there were more overall visits, then more visits for the row would have been needed than the 117.

> Unique page-views changed from dropped from 6510 in Mar of 2022 to 2335 (when?).

That could be shown in the future but would require code changes. IT's basically shown in the date selector. In an ideal world it would say it directly in there.

> Number behind movers. Whatever (number) is, why isn't it there for movers?

Sorry not sure what you meant here?

Hope this helps.

Reply:
@tsteur
I changed my proposal to how I currently understand it.
![bilde](https://user-images.githubusercontent.com/13802408/176593747-b61522eb-cc20-4fdc-88b8-722ee9f266ec.png)
isn't quite the same string, so there is room for error.